### ISSUE TITLE
[Broken] Atresplayer

### ISSUE THREAD

**Issue by heroku-miraheze** [yt-dlp/yt-dlp#996]
## Checklist

- [x] I'm reporting a broken site support
- [x] I've verified that I'm running yt-dlp version **2021.09.02**
- [x] I've checked that all provided URLs are alive and playable in a browser
- [x] I've checked that all URLs and arguments with special characters are properly quoted or escaped
- [x] I've searched the bugtracker for similar issues including closed ones

## Verbose log

```
C:\Users\info>yt-dlp -u (PRIVATE) -p (PRIVATE) https://www.atresplayer.com/kidz/series-infantiles/little-charmers/temporada-1/capitulo-1-un-principe-poco-encantador-el-vestido-encantado_602e67b17ed1a8453038d061/ -v
[debug] Command-line config: ['-u', 'PRIVATE', '-p', 'PRIVATE', 'https://www.atresplayer.com/kidz/series-infantiles/little-charmers/temporada-1/capitulo-1-un-principe-poco-encantador-el-vestido-encantado_602e67b17ed1a8453038d061/', '-v']
[debug] Encodings: locale cp1252, fs utf-8, out utf-8, pref cp1252
[debug] yt-dlp version 2021.09.02 (exe)
[debug] Python version 3.8.10 (CPython 64bit) - Windows-10-10.0.19043-SP0
[debug] exe versions: none
[debug] Optional libraries: mutagen, pycryptodome, sqlite, websockets
[debug] Proxy map: {}
[AtresPlayer] Downloading login page
ERROR: [AtresPlayer] 602e67b17ed1a8453038d061: Unable to download webpage: HTTP Error 404: Not Found (caused by <HTTPError 404: 'Not Found'>); please report this issue on  https://github.com/yt-dlp/yt-dlp . Make sure you are using the latest version; see  https://github.com/yt-dlp/yt-dlp  on how to update. Be sure to call yt-dlp with the --verbose flag and include its complete output.
  File "yt_dlp\extractor\common.py", line 693, in _request_webpage
  File "yt_dlp\YoutubeDL.py", line 3218, in urlopen
  File "urllib\request.py", line 531, in open
  File "urllib\request.py", line 640, in http_response
  File "urllib\request.py", line 569, in error
  File "urllib\request.py", line 502, in _call_chain
  File "urllib\request.py", line 649, in http_error_default

```

## Description

Atresplayer not working.

**Comment by Ashish0804** [comment]
Edit your message to include verbose log,

**Comment by heroku-miraheze** [comment]
@Ashish0804 Yes, fixed.

**Comment by coletdjnz** [comment]
Possibly fixed by https://github.com/ytdl-org/youtube-dl/pull/27215?

**Comment by heroku-miraheze** [comment]
Yes. @coletdjnz

**Comment by coletdjnz** [comment]
I've pinged the dev to see if they want to make the PR here

**Comment by felisucoibi** [comment]
please include this pr in master

**Comment by pukkandan** [comment]
The PR author seems to have abandoned it.

https://github.com/yt-dlp/yt-dlp/pull/1165#issuecomment-962490930 claims that the site has content that can be accessed without an account, but the links in that comment do ask for login. Does anyone have a link that actually doesn't need login? If not, you can send account details to me via discord (pukkandan#4207) / reddit (/u/pukkandan) / email (pukkandan.ytdlp@gmail.com) if u want me to have a look

**Comment by felisucoibi** [comment]
Yes, the website nows needs a cookie.txt with auth credentials confirmed to wrok, and almost everything is paid content. This pr works on soft protected links, not with hard proteced, that means half of the content can be donwnlaoaded with a paid cooke.txt, also i think it uses geoblocking, but anyway the pr works. I'll try to contact you on discord.

**Comment by joaquinito2070** [comment]
> The PR author seems to have abandoned it.
> 
> [#1165 (comment)](https://github.com/yt-dlp/yt-dlp/pull/1165#issuecomment-962490930) claims that the site has content that can be accessed without an account, but the links in that comment do ask for login. Does anyone have a link that actually doesn't need login? If not, you can send account details to me via discord (pukkandan#4207) / reddit (/u/pukkandan) / email ([pukkandan.ytdlp@gmail.com](mailto:pukkandan.ytdlp@gmail.com)) if u want me to have a look

@pukkandan I mentioned my Atresplayer credentials to your Discord.

**Comment by pukkandan** [comment]
It's been a long time since that comment. I don't have the time or interest to deal with this myself anymore. Some other dev will have to take it up.

**Comment by mmoya** [comment]
Confirming for me the issue is fixed with #1165:

```
cd /path/to/checkout/yt-dlp
git fetch refs/pull/1165:PR-1165
git checkout PR-1165
./yt-dlp.sh -u user@example.com -p secret https://www.atresplayer.com/...
```

**Comment by felisucoibi** [comment]
> Confirming for me the issue is fixed with #1165:
> 
> ```
> cd /path/to/checkout/yt-dlp
> git fetch refs/pull/1165:PR-1165
> git checkout PR-1165
> ./yt-dlp.sh -u user@example.com -p secret https://www.atresplayer.com/...
> ```

did you try subtitles? they worked but i think this week they stopped working.

**Comment by mmoya** [comment]
> did you try subtitles? they worked but i think this week they stopped working.

No, just downloaded a video.

**Comment by bashonly** [comment]
For a more modern log, see #8359
